NEWS.md

```diff
@@ -1,4 +1,13 @@
 # ggplot2 3.1.0.9000
+
+* `geom_hline()`, `geom_vline()` & `geom_abline()` now throw a warning if the user supplies both an xintercept, yintercept or slope value & a mapping such as this:
+  
+  ```R
+  geom_hline(yintercept = 3, aes(colour = colour))
+ 
+  ```
+  
+  The above will cause the user supplied mapping to be ignored/overwritten and as users might resaonably expect the above to work given that other geoms do  not have this behaviour, they will now be warned about mapping being overwritten. (@RichardJActon, #2950)
```

Comment:
This news item seems overly verbose. Something like the first sentence (on line 3) would be sufficient. Also, please write "and" instead of "&" throughout, and please use the Oxford comma: `` `geom_hline()`, `geom_vline()`, and `geom_abline()` ``. Finally, please place the final period after `(@RichardJActon, #2950)`, not before.

Reply:
Two more comments on the same topic:
1. `xintercept`, `yintercept`, etc should be enclosed in backticks in the text.
2. The Oxford comma also applies to "or".
 
I.e.: `` `xintercept`, `yintercept`, or `slope` ``